CenterofExcellenceALMAccelerator/SolutionPackage/Other/Customizations.xml

```diff
@@ -24,13 +24,30 @@
       <statecode>0</statecode>
       <statuscode>1</statuscode>
     </connectionreference>
+    <connectionreference connectionreferencelogicalname="cat_sharedcat5fcustomazuredevops5f98780803e0b89f2d_0d7e2">
+      <connectionreferencedisplayname>CustomAzureDevOps CenterofExcellenceALMAcceleratorAzureDevOps-0d7e2</connectionreferencedisplayname>
+      <connectorid>/providers/Microsoft.PowerApps/apis/shared_cat-5fcustomazuredevops-5f312a92a6a1e366f9</connectorid>
+      <customconnectorid>
+        <connectorid>37284777-a05f-4680-b0f9-1eb13e90fc71</connectorid>
+      </customconnectorid>
+      <iscustomizable>1</iscustomizable>
+      <statecode>0</statecode>
+      <statuscode>1</statuscode>
+    </connectionreference>
     <connectionreference connectionreferencelogicalname="cat_sharedcommondataservice_b1aba">
       <connectionreferencedisplayname>Microsoft Dataverse (legacy) CenterofExcellenceALMAccelerator-b1aba</connectionreferencedisplayname>
       <connectorid>/providers/Microsoft.PowerApps/apis/shared_commondataservice</connectorid>
       <iscustomizable>1</iscustomizable>
       <statecode>0</statecode>
       <statuscode>1</statuscode>
     </connectionreference>
+    <connectionreference connectionreferencelogicalname="cat_sharedcommondataserviceforapps_76cb1">
+      <connectionreferencedisplayname>Microsoft Dataverse CenterofExcellenceALMAccelerator-76cb1</connectionreferencedisplayname>
+      <connectorid>/providers/Microsoft.PowerApps/apis/shared_commondataserviceforapps</connectorid>
```

Comment:
Is this connection reference a duplicate of cat_sharedcommondataservice_b1aba or is the non-Legacy dataverse connector? If it's a duplicate then we should update the reference for this to point to the existing connection reference to avoid users having to create multiple connections.

Reply:
Feel free to merge this if the above feedback isn't relevant or if you fix it. Everything else looks good.

Reply:
This is the modern/native connector for dataverse.